frontend/apps/filemanager/filemanager.lua

```diff
@@ -1098,8 +1089,28 @@ function FileManager:createFolder()
     input_dialog:onShowKeyboard()
 end
 
-function FileManager:deleteFile(file)
-    local ok, err, is_dir
+function FileManager:deleteFileDialog(file, post_delete_callback, pre_delete_callback)
```

Comment:
As methods are verbs, I would read that as "a method that deletes the file dialog" :)
Better be `:showDeleteFileDialog()` then :)